### PR TITLE
feat: LT-1289 - Picklist help

### PR DESCRIPTION
**Pull request by jfaracik**
* Fixed styling for users/picklists page
* Added text highlighting back
* Moved over to SingleFormViews
* Split the letter paragraphs forms out for picklist items
* Added help modals to form

NOTE: It has been okayed with content that help information can stay in `markdown-help.html` due to the quantity of it, as well as it being easier to update in the future due to its HTML nature.

Screenshot 2020-02-06 at 15 24 50
Screenshot 2020-02-06 at 15 25 02

### CODE REVIEW

File: picklists/forms.py [hunk 1]
```diff
@@ -1,29 +1,21 @@
...
-
 
 def add_picklist_item_form(request):
     picklist_type = request.GET.get("type")
 
     return Form(
         title=getattr(strings.Picklist.Create, picklist_type.upper()),
         questions=[
-            _name,
+            TextInput(title="Name", name="name"),
```
Comment (Mark-Spiers-kainos) [inline picklists/forms.py]: please moves the titles to lite content

File: templates/teams/markdown-help.html [hunk 2]
```diff
@@ -0,0 +1,205 @@
+{% load static %}
+
+<a id="link-help-formatting" class="app-markdown__help-link" href="#">
+	Help with formatting
+</a>
+<a id="link-help-variables" class="app-markdown__help-link" href="#">
+	What variables can I use to personalise my text?
```
Comment (Mark-Spiers-kainos) [inline templates/teams/markdown-help.html]: again lite content

File: templates/teams/markdown-help.html [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+						{% endfor %}
+					</tbody>
+				</table>
+			</div>
+		</div>
+	</div>
+</div>
+
+<script src="{% static 'javascripts/modal.js' %}"></script>
+<script type="text/javascript">
```
Comment (Mark-Spiers-kainos) [inline templates/teams/markdown-help.html]: can we move the js out of the file?

File: templates/teams/picklist.html [hunk 4]
```diff
@@ -3,67 +3,65 @@
...
+				<li class="lite-subnav__link {{ 'standard_advice'|add_subnav_selected_class:request.get_full_path }}">
+					<a class="govuk-link govuk-link--no-visited-state" href="{% url 'picklists:picklists' %}?type=standard_advice">Standard Advice</a>
+				</li>
+				<li class="lite-subnav__link {{ 'footnotes'|add_subnav_selected_class:request.get_full_path }}">
+					<a class="govuk-link govuk-link--no-visited-state" href="{% url 'picklists:picklists' %}?type=footnotes">Footnotes</a>
+				</li>
+			</ul>
+		</div>
+		<div class="govuk-grid-column-three-quarters">
+			<div style="margin: auto; max-width: 700px;">
```
Comment (Mark-Spiers-kainos) [inline templates/teams/picklist.html]: please remove inline style

File: templates/teams/markdown-help.html (hunk 2 above)
Comment (jfaracik) [reply templates/teams/markdown-help.html]: As mentioned above this page has been okayed by content to not be in the content system.

File: templates/teams/markdown-help.html (hunk 3 above)
Comment (jfaracik) [reply templates/teams/markdown-help.html]: What benefit would that provide?

File: letter_templates/context_variables.py [hunk 5]
```diff
@@ -87,4 +87,4 @@ def get_sample_context_variables():
 
 
 def get_key_value_pair(data):
-    return [{"key": item, "representation": "{" + item + "}}"} for item in data]
+    return [{"key": item, "representation": "{ " + item + " }}"} for item in data]
```
Comment (a-bek) [inline letter_templates/context_variables.py]: What do you think of using f strings from now?

File: picklists/forms.py [hunk 6]
```diff
@@ -49,9 +42,9 @@ def edit_picklist_item_form(picklist_item):
     return Form(
         title="Edit " + picklist_item["name"],
         questions=[
-            _name,
```
Comment (a-bek) [inline picklists/forms.py]: 👍

**Pull request merged**